### ISSUE TITLE
[cli] add `add` subcommand

### ISSUE THREAD

**Issue by miguelrk** [netzo/netzo#26]
**TLDR:** Add a `netzo add` subcommand to the CLI, which asks you which `component/block` you want to scaffold to your project, and the CLI takes care of creating the necessary files in `components/`, for example `components/data-table/`. This subcommand should be done implemented using the versatile [feathershq/pinion](https://github.com/feathershq/pinion).

### Context

Copy-pasting ready-made building block (built using more elementary/atomic UI components) is great. But copy paste sometimes involves creating multiple files, knowing where to put them, etc. Especially for more complex components such as a DataTable, which uses buttons, search inputs, dropdowns, the table, etc. There it is not only that you copy-paste the giant DataTable building-block to a single file, but rather you copy-paste several files...

### Implementation

Analogous to the `netzo init` subcommand which already implements the logic for scaffolding, the `netzo add` subcommand should grab the URL pointing to the `component/block` on GitHub and clone it by default to `.components/` (when you run the subcommand from the root of your fresh project). Also, the required `imports` for the component should be added to `deno.json`. To update the `deno.json` config file we could do this directly by reading file via `const denoJson = JSON.parse(await Deno.readTextFile("./deno.json"))` and mutating `denoJson` directly using utils like `merge`, etc or use [unjs/magicast](https://github.com/unjs/magicast) alternatively.

**Comment by miguelrk** [comment]
Closed by https://github.com/netzo/netzo/pull/165 for now
